### DISCUSSION TITLE
Et Futurum Requiem 2.6.1: It's About Time

### DISCUSSION THREAD

**Discussion by Roadhog360** [Roadhog360/Et-Futurum-Requiem#544] [New Release]
## Major Changes:
- **Now requires UniMixins** (GTNH Mixins module MUST be present)
- Mixins have been moved to use the new early and late mixins system.
- Internally updated recipes to check for GTNH

Additions:
- [Option to make some dimensions be entirely deepslate](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/62a3adacddeefcee696dafb535241d06e7d52abd) (PR [#515](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/515))
- [Option to disable bamboo generation but keep the blocks](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/3a414540fb6987ad7d51f9469077afe26b690251) (PR [#507](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/507))
- Option to enable fences, pressure plates, buttons etc for newly added wood SEPARATELY from the ones for vanilla woods (PR [#503](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/503))
- Option to make deepslate hardness the same as stone (PR [#501](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/501))
- [New tuff variants, copper variants, copper doors, trapdoors and copper bulbs](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/50e85324167470b6883535a56654c9e18f89db49)
- [Added `playersSleepingPercentage` gamerule](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/d1e93e5f20263172e44425eceac5a2231ef4b4fc) (PR [#537](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/537))

Fixes:
- [Possibly fix ConcurrentModificationException crash with GTNH NEI and furnace fuel handler](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/d51b9ebfa190255b0841e907aeb8b73eec7dcabb)
- [Fix signs pick block crash](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/3c6fd4c943b17bc17303fde3dc990f3de313521a)
- [Fix severe crashing with TCon smelteries when certain mods or config options are used](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/ede416781cb36c8215ef430d704d8d848a85554b)
- [Option for 3D bed models was ignored](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/25c8b332c50a610e5828a90a9f71e7ef9ee5b786)
- [Fixed falling dust particles falling from wrong spot in blocks](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/1e4088fc1015a59fcd13ed57cc06590fcc1bb8eb)
- [Fixed trapdoors having stone break sound](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/bfb7ffdc59f2d04c54739e9807877f91effac4a9) (Issue [#488](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/488))
- [Fixed Shulker Boxes being stackable inside of each other](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/b8b1a5c4718c4e8e0aca6fd5e19349f3e6a753f3) (Issue [#487](https://github.com/Roadhog360/Et-Futurum-Requiem/issues/487))
- [Fixed muddy mangrove roots (experimental) not supporting some plants](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/4f5d01277862c9d5fa2ad08bbf067baeabad8f41)
- [Fixed many entity interactions in spectator mode](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/5e4c1201d3bb400e62a64d4e918b675225335420)
- [Fixed spectators being counted when calculating sleeping players](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/4024f1675dc3cb067f562e71266ee5177ca475ed)
- [Fixed concrete logic running on the client](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/e25a52ddf31c98108ea61d2ed1eb791a971f81db)
- [Fixed slime pistons and possibly many more not working due to a registry object having incorrect hashing](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/64bce9eabab8e10312c2f577a1afaf7ea8513b5a)

Misc Changes:
- [Use IExtendedEntityProperties instead of a separate file for saving enchantment seeds](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/c7b02735f94aff96bff89e06acbecc297f497fa4) (PR [#531](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/531))
- [Additional blocks can be used as beehive smokers](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/fad3ef4815d591da9601f696f43a02e3b780dc0d) (`"Thaumcraft:blockAiry"`, `mod_ecru_MapleTree:ecru_BlockFallenLeavesFire`, `BambooMod:campfire`, `campfirebackport:foxfire_campfire`, `campfirebackport:shadow_campfire`)
- [Grass inventory item is now colored](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/d2c92ab0e9cf3ffb744a6059737ae63c0a51e725) (PR [#530](https://github.com/Roadhog360/Et-Futurum-Requiem/pull/530))
- [Revised BluePower/ProjectRed smooth stone compat](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/6e423f8d6c271c887fa52f200d69614501a32ceb)
- Updated fast Cherry Leaves texture to use the texture from bedrock edition
- [If all enchanting fuels are removed from the fuels list, then no fuels are required and the GUI switches to the 1.7 one](https://github.com/Roadhog360/Et-Futurum-Requiem/commit/8c52c3ae5c6d830a6fe56b20c6628e254e1a0971) (but keeps 1.8 updated values)

And of course, thanks to kotmatross28729 and omgise for consistent translation updates!

## The following is from experimental content, only available in `experiments.cfg`. This content may be unfinished and/or broken.

Additions:
- Started work on bubble columns

<hr /><em>This discussion was created from the release <a href='https://github.com/Roadhog360/Et-Futurum-Requiem/releases/tag/2.6.1'>Et Futurum Requiem 2.6.1: It's About Time</a>.</em>

**Comment by reallydrained** [comment]
Seeing there was an update today made me SO happy =D I love this project, it's the single most important mod in my singleplayer world and the server I host for me and my friends. Having working rocket elytra flight with backlytra is AMAZING (just saying!) and adding the gamerule playersSleepingPercentage made it so I can remove a mod I had just for that issue.

If I had to request one thing it would be adding offhand (left hand) item holding slot (if that is even possible, just for totems) -- PLEASE keep up the amazing work bro, there are people like me that appreciate what you're doing SO much. 1.7.10 forever 😎

**Reply by Roadhog360** [reply]
Check out [Backhand](https://github.com/GTNewHorizons/Backhand/releases/tag/1.5.12)!

**Reply by reallydrained** [reply]
Oh, awesome!! Thank you =))
